tree/treeplayer/inc/ROOT/TTreeProcessorMT.hxx

```diff
@@ -46,6 +48,13 @@ the threaded object.
 */
 
 namespace ROOT {
+
+/// A cluster of entries
+struct EntryRange {
+   Long64_t start;
```

Comment:
Also here I wonder whether we can make these `ULong64_t`

Reply:
Similar to above, yes we can. But either means casts, or refractoring TTreeProcessorMT. And I am unsure if refractoring TTreeProcessorMT is possible, since it makes calls to the TTreeReader, which keeps `Long64_t` since `-1` for instance has a special meaning